ios/Pods/MobileCenter/MobileCenter-SDK-iOS/MobileCenter.framework/Headers/MSConstants.h

```diff
@@ -0,0 +1,47 @@
+#import <Foundation/Foundation.h>
```

Comment:
We did consider including all of that in the `mobile-center-analytics` or `mobile-center-crashes` package, so that the user just installs it from node_modules. Infact, thats what we do with one zip library in codepush.
However, the modules that we use are pretty popular and we found that there are many other libraries that use them too - causing conflicts. CodePush has had similar issues - that's why we choose to go the Cocoapods (and Carthage in the future) route. You technically don't have to worry about installing Cocoapods or downloading the packages. Cocoapods sort of recommends checking in the native changes.
The alternative would be to manually add these frameworks into the project.